lib/Site.js

```diff
@@ -204,6 +205,47 @@ Site.prototype.buildSourceFiles = function () {
   });
 };
 
+/**
+ * Rebuild rebuild source files that are affected by change in filePath
```

Comment:
- Repeated `rebuild`.
- Can you add jsdoc to `buildSourceFiles` and `generatePages` as well, since these are related?

Reply:
Will do!